packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -162,7 +162,11 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
 ///
 // See https://github.com/flutter/flutter/issues/54059
```

Comment:
Why would this issue still be here? Isn't this PR meant to resolve the issue?

Reply:
Sorry, I missed this place. I have already deleted it now.